NOnion/Cells/CellCreate2.fs

```diff
@@ -0,0 +1,32 @@
+﻿namespace NOnion.Cells
+
+open System.IO
+
+open NOnion.Utility
+
+type CellCreate2 =
```

Comment:
why does this have a 2 suffix?

Reply:
That's the name of the cell in spec and we should be inline with it, There is a create cell as well but I didn't implement it yet because it's legacy and only for backward compatibility with old routers

Reply:
let's add a comment with a link to the spec where it mentions this weird cells with the 2 suffix